relay-server/src/actors/processor.rs

```diff
@@ -2082,7 +2082,7 @@ impl EnvelopeProcessorService {
             }
 
             if let Some(config) = config {
-                let metrics = crate::metrics_extraction::generic::extract_metrics(event, config);
+                let metrics = crate::metrics_extraction::event::extract_metrics(event, config);
```

Comment:
Note how there is now just one entry point for extracting metrics from an event: The event function. The idea is that there is one top-level extraction function for every data category that is able to traverse data internally. These functions will mostly need to forward to the generic extract function.

I'm planning a follow-up that makes the generic function a provided trait method to solidify this design.